docs/source/misc/changelog.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.1.16] - 2023-02-06
+
+### Added
+- RankMatchFailure metric for evaluation
+- Statistical significance and power analysis utilities
+- Stat analysis for groupwise metrics in Ranking
+
```

Comment:
@mohazahran I've updated the changelog to capture all changes we will be publishing in this new version.